frontend/src/scenes/session-recordings/playlist/sessionRecordingsPlaylistLogic.ts

```diff
@@ -468,13 +468,18 @@ export const sessionRecordingsPlaylistLogic = kea<sessionRecordingsPlaylistLogic
             { persist: true, prefix: `${getCurrentTeamId()}__${key}` },
             {
                 setFilters: (state, { filters }) => {
-                    if (!isValidRecordingFilters(filters)) {
-                        console.error('Invalid filters provided:', filters)
-                        return state
-                    }
-                    return {
-                        ...state,
-                        ...filters,
+                    try {
+                        if (!isValidRecordingFilters(filters)) {
+                            console.error('Invalid filters provided:', filters)
+                            return getDefaultFilters(props.personUUID)
```

Comment:
i think we should still return `state` here... i.e. "you tried to change these and we will not"

and then reset to default in the catch block

but not a strong opinion

Reply:
LGTM, (still think state is slightly more correct here but 🤷) :)